packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -55,24 +54,15 @@ let client: LanguageClient;
 let statusBarItem: StatusBarItem;
 let currentModel: UI5Model | undefined;
 
-export async function activate(context: ExtensionContext): Promise<void> {
-  // create the LanguageClient (+Server)
-  client = createLanguageClient(context);
-
-  // register semantic token provider
-  context.subscriptions.push(
-    languages.registerDocumentSemanticTokensProvider(
-      { language: "xml" },
-      bindingSemanticTokensProvider,
-      bindingLegend
-    )
-  );
-
+async function init(context: ExtensionContext): Promise<void> {
```

Comment:
Why is this function async? There are no awaits and it does not return a promise.